test/notifications/notification_server_test.exs

```diff
@@ -369,7 +369,7 @@ defmodule Notifications.NotificationServerTest do
         end)
 
       assert_n_notifications_in_db(1)
-      assert log == ""
+      assert String.contains?(log, "new_notification") == false
```

Comment:
Is there a reason to use `String.contains?(...)` rather than `assert log =~ "thing we don't want"`?

Reply:
Sorry, missing a `not` in there.